l10n_fi_business_code/data/res_partner_id_category.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
This is Odoo 10, please use `<odoo>` and in cases of `<openerp><data noupdate="1">` use `<odoo noupdate="1">`